cfme/infrastructure/virtual_machines.py

```diff
@@ -354,39 +354,64 @@ def does_snapshot_exist(self):
             self._nav_to_snapshot_mgmt()
             try:
                 if self.name is not None:
-                    self.snapshot_tree.find_path_to(re.compile(self.name + r".*?"))
+                    self.snapshot_tree.find_path_to(re.compile(r"{}.*?".format(self.name)))
                 else:
-                    self.snapshot_tree.find_path_to(re.compile(self.description + r".*?"))
+                    self.snapshot_tree.find_path_to(re.compile(r"{}.*?".format(self.description)))
                 return True
             except CandidateNotFound:
                 return False
             except NoSuchElementException:
                 return False
 
-        def wait_for_snapshot_active(self):
-            self._nav_to_snapshot_mgmt()
+        def _snapshot_click_helper(self, prop):
+            """
+            Helper method to reduce code duplication.
```

Comment:
REQUIRED: we use docstrings where the summary line is on the same line as the opening quotes. Please use that as well for consistency.